restore/restore_internal_test.go

```diff
@@ -129,6 +174,51 @@ var _ = Describe("restore internal tests", func() {
 					Schema: "foo2", Name: "foopart_p1", ObjectType: toc.OBJ_TABLE, ReferenceObject: "foo2.foopart",
 					Statement: "\n\nALTER TABLE ONLY foo2.foopart ATTACH PARTITION foo2.foopart_p1 FOR VALUES FROM (0) TO (1);\n",
 				},
+				{
+					Schema: "foo2", Name: "bar", ObjectType: toc.OBJ_STATISTICS,
+					Statement: `DELETE FROM pg_statistic WHERE (starelid, staattnum) IN
+(SELECT attrelid, attnum FROM pg_attribute WHERE attrelid = 'foo2.bar'::regclass::oid AND attname = 'i');`,
```

Comment:
lines 177-180 are repeated at 193-195 and 208-210. Other lines are repeated too. Why do we need this?

Reply:
These are the expected results after replacing the schema to `foo2` in data from the `statements` above them. And the input data for all these extected results is different.
So, in other words we are checking that
```
foo.bar -> foo2.bar
"fo.o".bar -> foo2.bar
"fo'o".bar -> foo2.bar
```
replacements work correctly.